src/components/commentary/CommentaryCombo.js

```diff
@@ -83,6 +84,10 @@ const useStyles = makeStyles((theme) => ({
     fontSize: "1rem",
     cursor: "pointer",
   },
+  versionSelected: {
+    boxShadow: "inset 0 0 30px " + LIGHTGREY,
+    border: "1px solid #ccc",
```

Comment:
We can make it more noticeable by adding a grey border with 70 opacity. 
We can do the same for version.js also, Note Grey has to be imported
```suggestion
    border: "1px solid " + GREY + "70",
```

Reply:
Added GREY border with 70 opacity.
Also, did the same changes in Version.js file